Templates/Tests/Generated/AutoMockable.generated.swift

```diff
@@ -423,6 +423,90 @@ class AnyProtocolMock: AnyProtocol {
     }
 
 
+}
+class AnyProtocolWithOptionalsMock: AnyProtocolWithOptionals {
+
+
+    var a: [(any StubProtocol])?
+    var b: [Result<Void, any Error>] = []
+    var c: (Int, [((any StubProtocol))?]))?
```

Comment:
`any StubProtocol` has two sets of parentheses + there's one too many at the property definition elvel